pages/common/tldr.md

```diff
@@ -12,7 +12,7 @@
 
 `tldr {{command}} {{subcommand}}`
 
-- Print the tldr page for a command in the given [L]anguage (if available, otherwise fall back to English):
+- Print the tldr page for a command in the given [l]anguage (if available, otherwise fall back to English):
```

Comment:
```suggestion
- Print the tldr page for a command in the given [L]anguage (if available, otherwise fall back to English):
```
The mnemonic capitalization is right here as it is used to highlight the short option for specifying a language in clients. The other change below for the list option looks fine.